crates/bevy_solari/src/realtime/gbuffer_utils.wgsl

```diff
@@ -0,0 +1,57 @@
+#define_import_path bevy_solari::gbuffer_utils
+
+#import bevy_pbr::pbr_deferred_types::unpack_24bit_normal
+#import bevy_pbr::rgb9e5::rgb9e5_to_vec3_
+#import bevy_pbr::utils::octahedral_decode
+#import bevy_render::view::View
+#import bevy_solari::scene_bindings::ResolvedMaterial
+
+struct ResolvedGPixel {
+    world_position: vec3<f32>,
+    world_normal: vec3<f32>,
+    material: ResolvedMaterial,
+}
+
+fn gpixel_resolve(gpixel: vec4<u32>, depth: f32, pixel_id: vec2<u32>, view_size: vec2<f32>, world_from_clip: mat4x4<f32>) -> ResolvedGPixel {
+    let world_position = reconstruct_world_position(pixel_id, depth, view_size, world_from_clip);
+    let world_normal = octahedral_decode(unpack_24bit_normal(gpixel.a));
+
+    let base_rough = unpack4x8unorm(gpixel.r);
+    let base_color = pow(base_rough.rgb, vec3(2.2));
```

Comment:
do we have no helper function that unpacks the gbuffer and does these conversions?

Reply:
That's what this file is for :P (but no)

Reply:
I think it should live in bevy_pbr then, as thats where the gbuffer is defined right